metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -301,13 +301,13 @@ def check_base_folder_if_not_provided(
     ) -> Optional[pydantic.DirectoryPath]:
         if v is None:
             git_info: Optional[GitInfo] = values.get("git_info")
-            if git_info and git_info.deploy_key:
-                # We have git_info populated correctly, base folder is not needed
-                pass
+            if git_info:
+                if not git_info.deploy_key:
```

Comment:
Small nit: I'd probably prefer two if statementst (`if git info and not git_info.deploy_key:`, `elif not git_info:`) but not a big deal either way

Reply:
yeah that'd probably be cleaner

will do in a follow up since I don't want to wait for CI again